Examples/Examples/All Examples/TrackingModeExample.swift

```diff
@@ -0,0 +1,59 @@
+import UIKit
+import MapboxMaps
+import MapboxCoreMaps
+import MapboxCommon
+
+@objc(TrackingModeExample)
+
+public class TrackingModeExample: UIViewController, ExampleProtocol {
+
+    internal var mapView: MapView!
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+
+        mapView = MapView(with: view.bounds, resourceOptions: resourceOptions())
+        self.view.addSubview(mapView)
+
+        // Set the map's style
+        mapView.style.styleURL = StyleURL.streets
+
+        // Add user position icon to the map with location indicator layer
+        mapView.update { (mapOptions) in
+            mapOptions.location.showUserLocation = true
+        }
+
+        // Set initial camera settings
+        let startingCoordinate = CLLocationCoordinate2D(latitude: 42.3601, longitude: -71.0589)
+        mapView.cameraManager.setCamera(centerCoordinate: startingCoordinate,
+                                                   zoom: 15.0)
+
+        // Create class which conforms to LocationConsumer, update the camera's centerCoordinate when a locationUpdate is received
+        class CameraLocationConsumer: LocationConsumer {
```

Comment:
Let's make this class something outside the scope of `public class TrackingModeExample`

We don't want to embed a class inside of another class, and definitely not inside of a function

Reply:
Resolved in most recent commit after pairing.